controllers/appointment.controller.js

```diff
@@ -0,0 +1,98 @@
+const sendEmail = require("../helpers/email");
+const appointmentService = require("../services/appointment.services");
+const authService = require("../services/auth.services");
+const { appointmentSchema } = require("../schemas/appointment");
+const { resolveError } = require("../helpers/controllers");
+
+const create = async (req, res) => {
+  try {
+    const authorizationHeader = req.headers.authorization;
+    const token = authorizationHeader.split(" ")[1];
+    const { id } = await authService.verifyToken(token);
```

Comment:
Your current user id is in the req object https://github.com/basicem/medica-api/pull/11/files#diff-f30160aefa12ca4840ebd31571a3ba802d37dbd8b00cffa0a0a0387adcf738f6L12

Reply:
?

Reply:
fixed it